tests/certificate/certificate.mutation.test.js

```diff
@@ -16,28 +16,31 @@ const {
 
 let operations;
 let certificateId;
+let certificateName;
 let isUsed;
 
-describe('Test mutation methods', () => {
+describe('Test mutation methods Admin', () => {
   beforeAll(async () => {
     operations = await setupApp();
   });
 
-  it('#1. create certificate', async () => {
-    const result = await addCertificate(newCertificateInputData, operations);
+  it('#1. Certificate was successfully Generated', async () => {
```

Comment:
use should, like in other cases

Reply:
ok, but https://stackoverflow.com/a/56072272/449347